contracts/contracts/sbtc-registry.clar

```diff
@@ -6,6 +6,13 @@
 (define-constant ERR_INVALID_REQUEST_ID (err u401))
 (define-constant ERR_AGG_PUBKEY_REPLAY (err u402))
 (define-constant ERR_MULTI_SIG_REPLAY (err u403))
+(define-constant ERR_INVALID_PROTOCOL_ID (err u404))
+
+
+;; protocol contract type
+(define-constant governance 0x00)
+(define-constant deposit 0x01)
+(define-constant withdrawal 0x02)
```

Comment:
```suggestion
(define-constant governance-role 0x00)
(define-constant deposit-role 0x01)
(define-constant withdrawal-role 0x02)
```

I just feel like this is a little more readable when used elsewhere.

Reply:
Updated